handlers.go

```diff
@@ -32,16 +32,19 @@ func mainHandler(w http.ResponseWriter, r *http.Request) {
 	var falcopayload types.FalcoPayload
 
 	stats.Requests.Add("total", 1)
+	countMetric("inputs", 1, []string{})
```

Comment:
in your PR comment you say `falcosidekick.incoming` and here it's `inputs`. In `expvar` I use `total`.

Reply:
Changed this to `total` to match the expvars.